core/arch/arm/kernel/entry_a32.S

```diff
@@ -253,7 +253,7 @@ DECLARE_KEEP_INIT _start
 		 * equivalent of:
 		 * extern const u32 stack_tmp_stride;
 		 * u32 stack_tmp_stride_rel = (u32)&stack_tmp_stride -
-		 *			      (u32)&stack_tmp_stride_rel 
```

Comment:
unrelated change. Prefer in a dedicated commit.

Reply:
Was commited by mistake. Removed the change